models/annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -93,328 +160,75 @@
-       "0             Sand  20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "1             Sand  20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "2             Sand  20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "3             Sand  20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "4             Sand  20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "...            ...                                                ...   \n",
-       "156367  Vegetation  20230910_105008_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "156368  Vegetation  20230910_105008_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "156369  Vegetation  20230910_105008_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "156370  Vegetation  20230910_105008_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "156371  Vegetation  20230910_105008_PNEO-03_1_1_30cm_RD_12bit_RGBN...   \n",
-       "\n",
-       "                   date  season                     annotation_no  \n",
-       "0       20230513_104139  Spring           31_20230513_annotations  \n",
-       "1       20230513_104139  Spring           31_20230513_annotations  \n",
-       "2       20230513_104139  Spring           31_20230513_annotations  \n",
-       "3       20230513_104139  Spring           31_20230513_annotations  \n",
-       "4       20230513_104139  Spring           31_20230513_annotations  \n",
-       "...                 ...     ...                               ...  \n",
-       "156367  20230910_105008    Fall  30_Annotations_Coepelduynen_2023  \n",
-       "156368  20230910_105008    Fall  30_Annotations_Coepelduynen_2023  \n",
-       "156369  20230910_105008    Fall  30_Annotations_Coepelduynen_2023  \n",
-       "156370  20230910_105008    Fall  30_Annotations_Coepelduynen_2023  \n",
-       "156371  20230910_105008    Fall  30_Annotations_Coepelduynen_2023  \n",
-       "\n",
-       "[1018424 rows x 13 columns]"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "df = pd.read_parquet(annotated_pixels_filepath)\n",
-    "df"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 18,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "date\n",
-       "20230513_104139    392936\n",
-       "20230402_105321    156372\n",
-       "20230601_105710    156372\n",
-       "20230908_110020    156372\n",
-       "20230910_105008    156372\n",
-       "Name: count, dtype: int64"
-      ]
-     },
-     "execution_count": 8,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "# This is to give an indication of the amount of data points per image\n",
-    "df['date'].value_counts()"
+    "df['label'] = df['label'].str.replace(\"GRound\",\"Ground\" )"
```

Comment:
Is there a more elegant place where we could fix this? Such as correcting the typo in the label file?

Reply:
should be in the data preparation file

Reply:
moved to data preparation file

Reply:
I cannot find it in the data preparation file. Can you reference it here?

Reply:
in [transform_polygon_annotations_to_pixels.ipynb](](https://github.com/Provincie-Zuid-Holland/satellite-images-nso-datascience/pull/17/files/7e463ac3be5efaa8f5b0a6ab59f789e77a5d7df6#diff-7d910b9838dd9cd839d89a7032a692fc42f4fec264e76044c38184bd753eb87bhttps://github.com/Provincie-Zuid-Holland/satellite-images-nso-datascience/pull/17/files/7e463ac3be5efaa8f5b0a6ab59f789e77a5d7df6#diff-7d910b9838dd9cd839d89a7032a692fc42f4fec264e76044c38184bd753eb87b)